pkg/sql/ttl/ttljob/ttljob.go

```diff
@@ -40,35 +39,12 @@ import (
 	"github.com/cockroachdb/errors"
 )
 
-var (
-	defaultSelectBatchSize = settings.RegisterIntSetting(
-		settings.TenantWritable,
-		"sql.ttl.default_select_batch_size",
-		"default amount of rows to select in a single query during a TTL job",
-		500,
-		settings.PositiveInt,
-		settings.WithPublic)
-	defaultDeleteBatchSize = settings.RegisterIntSetting(
-		settings.TenantWritable,
-		"sql.ttl.default_delete_batch_size",
-		"default amount of rows to delete in a single query during a TTL job",
-		100,
-		settings.PositiveInt,
-		settings.WithPublic)
-	defaultDeleteRateLimit = settings.RegisterIntSetting(
-		settings.TenantWritable,
-		"sql.ttl.default_delete_rate_limit",
-		"default delete rate limit for all TTL jobs. Use 0 to signify no rate limit.",
-		0,
-		settings.NonNegativeInt,
-		settings.WithPublic)
-
-	jobEnabled = settings.RegisterBoolSetting(
-		settings.TenantWritable,
-		"sql.ttl.job.enabled",
-		"whether the TTL job is enabled",
-		true,
-		settings.WithPublic)
+var jobEnabled = settings.RegisterBoolSetting(
```

Comment:
Hmm, it feels a bit strange to not see all these settings grouped together. How would you feel about moving this to ttlbase as well? If that's not easy, maybe add a note indicating that settings for the job execution can be found in ttl base?

Reply:
I'll move it as well for consistency.